src/Microsoft.DotNet.Build.Tasks/Targets/partialfacades.targets

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Import Project="$(MSBuildThisFileDirectory)buildtoolsversion.targets" />
+
   <UsingTask TaskName="PrereleaseResolveNuGetPackageAssets" AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"/>
 
   <PropertyGroup>
-    <GenFacadesNuGetName>Microsoft.DotNet.BuildTools.ApiTools.1.0.0-prerelease</GenFacadesNuGetName>
+    <GenFacadesNuGetName>GenFacades.Desktop.1.0.0-experimental-$(BuildToolsRevisionNumber)</GenFacadesNuGetName>
```

Comment:
Presently it needs to be restored by whatever restores the build tools (just like it's done today).  Would you like the target to restore it instead?